### PR TITLE
Replace Website with Webapp, closes #269

### LINKED ISSUE #269: Replace Outdated Website
Replace the website with something that knows about the e.V. and fixes the sign up process.

### PR DESCRIPTION
**Pull request by nils-wisiol**
Adds a new web site built on vue.js / webpack.

IMHO, these are must-haves for deployment:

- [x] port doc pages from old website
- [ ] ~~mailing list sign up form~~
- [x] how to set password for non-dyn users?
- [x] remove /static/ route from `www`
- [x] add some information about SSE cooperation
- [x] squash
- [x] include sponsor's logo
- [x] make sure our logo does not get truncated on mobile
- [x] (auto?)squash again
- [x] make sure that compiling webapp works on the first try, even if volume not initialized, or change deployment mechanism altogether

These are nice-to-have, or further TODOs after this is merged:

- [ ] new logo
- [ ] webapp tests
- [x] password reset form
- [ ] change email from
- [ ] delete account form
- [ ] list of domains

Please feel free to edit/amend/update these lists.

### CODE REVIEW

**Comment by nils-wisiol** [comment]
I think the privacy policy got lost when rebasing. Will look into it soon. 🙄

**Comment by peterthomassen** [comment]
> I think the privacy policy got lost when rebasing. Will look into it soon.

It was just the router. fixed in e164398f69be6831af492283b3f16ffff8da136a

File: webapp/src/views/SignUp.vue [hunk 1]
```diff
@@ -113,7 +113,11 @@
                       tabindex="4"
               >
                 <template slot="label">
-                  <v-flex>Yes, I agree to the <a @click.stop="privacyPolicy">privacy policy</a>.</v-flex>
+                  <v-layout>
```
Comment (nils-wisiol) [inline webapp/src/views/SignUp.vue]: remove v-layout?

File: api/desecapi/tests/test_user_management.py [hunk 2]
```diff
@@ -412,8 +412,8 @@ def _test_registration_with_domain(self, email=None, password=None, domain=None,
         if self.has_local_suffix(domain):
             cm = self.requests_desec_domain_creation_auto_delegation(domain)
         else:
-            cm = self.requests_desec_domain_creation(domain)
-        with self.assertPdnsRequests(cm[:-1]):
+            cm = self.requests_desec_domain_creation(domain)[:-1]
```
Comment (nils-wisiol) [inline api/desecapi/tests/test_user_management.py]: see if there's a better way to achieve this

File: api/desecapi/views.py [hunk 3]
```diff
@@ -635,7 +634,8 @@ class AuthenticatedResetPasswordUserActionView(AuthenticatedActionView):
     serializer_class = serializers.AuthenticatedResetPasswordUserActionSerializer
 
     def finalize(self):
-        return Response({'detail': 'Success! Your password has been changed.'})
+        login_url = self.request.build_absolute_uri(reverse('v1:login'))
+        return Response({'detail': f'Success! Your password has been changed. You can now log in at {login_url}.'})
```
Comment (nils-wisiol) [inline api/desecapi/views.py]: remove "now"

**Pull request merged**